static/flow/flow-app.js

```diff
@@ -12,6 +12,9 @@ function initFlowApp() {
 
     // showTopLevelView('landing-page-view');
     showTopLevelView('login-page-view');
+    if(g_fullscreen) {
+        var fullscreenWidget = FullScreenWidget();
```

Comment:
`FullScreenWidget()` doesn't actually return anything, does it?

Reply:
Not at present.  No reason to store it in a variable.  Will fix!